tests/CoinTests.cpp

```diff
@@ -186,13 +186,65 @@ TEST(Coin, ValidateAddressDGB){
     EXPECT_FALSE(validateAddress(TWCoinTypeDigiByte, "SUngTA1vaC2E62mbnc81Mdos3TcvZHwsVx"));
 }
 
+TEST(Coin, validateAddressIocoin) {
+    EXPECT_TRUE(validateAddress(TWCoinTypeIocoin, "ik6V522aupkj3kMtJZi6eDw6v3eLgMbXY8"));
```

Comment:
Did you consider adding a new file: `CoinAddressValidationTests.cpp` ? just

Reply:
Done in 3351155e1177f17cc56d62cfeacfcf0ba26a70e2